test/testHTMLFromMarkdown.js

```diff
@@ -296,22 +289,21 @@ describe('Testing Markdown conversion', () => {
         # location
         <a name="anchors">Foo</a>
       `, `
-        <h1 id="location">location</h1>
+        <h1>location</h1>
```

Comment:
@ramboz I think it indeed should remove the location attribute, right?

Reply:
🤦‍♂ yes!